pkg/cmd/proxy.go

```diff
@@ -97,6 +98,7 @@ func setupProxyTunnel(opts *proxyOpts) error {
 				http.Error(w, err.Error(), code)
 			}
 		}),
+		ReadHeaderTimeout: 1 * time.Minute,
```

Comment:
While this fixes the linter failure, I'm not sure what's the "correct" value that we should use here. Different servers are using different values. For example, Nginx uses 60 seconds and that was the inspiration for choosing 1 minute.

Reply:
60 seconds sounds fine to me.